README.md

```diff
@@ -160,9 +162,6 @@ generateOpenAPI({
   packageDir: ".",
   outputFile: "openapi.json",
   pathGlob: "/src/pages/api/**/*.ts",
-
-  // Tags improve the organization of an OpenAPI spec by making "expandable"
-  // sections including routes
```

Comment:
why remove this?

Reply:
I see you added it to command line parameters, but it would need to be somewhere else or in it's own section or something

Reply:
Actually, I switched to the description above, to all parameters are described together.